src/components/GlobalFilter/globalFilterApi.test.js

```diff
@@ -33,8 +33,8 @@ describe('flatTags', () => {
 
   it('should create flat array of global filter', () => {
     const data = flatTags(globalFilter);
-    expect(data.length).toBe(2);
-    expect(data).toMatchObject(['someTag%2Fslash/someKey=[someValue%3Dvalue]', 'Workloads/SAP']);
+    expect(data.length).toBe(3);
+    expect(data).toEqual(expect.arrayContaining(['someTag%2Fslash/someKey=[someValue%3Dvalue]', 'Workloads/SAP', 'SAP ID (SID)/SOMEVAL']));
```

Comment:
Why did these change? Ideally, the output should be exactly the same after refactoring.

I just want to make sure we are not adapting the tests to a "buggy" code.